pkgs/development/python-modules/aiocron/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python
+, croniter
+, tzlocal
+, pytest
+, pytest-cov
+}:
+
+buildPythonPackage rec {
+  pname = "aiocron";
+  version = "1.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-rX07m9iIl5NP/RiTf/I1q5+wE1goD9QOAUYf1fdjSL0=";
+  };
+
+  propagatedBuildInputs = [
+    croniter
+  ];
+
+  checkInputs = [
+    pytest
+    pytest-cov
+    tzlocal
+  ];
+
+  checkPhase = ''
+    pytest tests/
+  '';
```

Comment:
This could probably be remove if you switch `pytest` to `pytestCheckHook` on line 25.